en/docs/setup/deployment-Patterns.md

```diff
@@ -2,68 +2,59 @@
+to set up and configure the deployment pattern. 
 
 ### Deployment prerequisites
 
 As a first step in planning your deployment, ensure that you have the
 necessary system requirements and a compatible environment.
 
-**System requirements**
-
-|        |                                                                                          |
-|--------|------------------------------------------------------------------------------------------|
-| Memory | **4GB** : 2 GB for the Java Virtual Machine (JVM) and 2 GB for the Operating System (OS) |
-| Disk   | 10 GB minimum                                                                            |
-
-**Environment compatibility**
-
+**System requirements** 
+<table>
+    <tr>
+        <th>Memory</th>            
+        <td><b>4GB</b>: 2 GB for the Java Virtual Machine (JVM) and 2 GB for the Operating System (OS)</td>
+    </tr>
+    <tr>
+        <th>Disk</th>            
+        <td>10 GB minimum</td>
+    </tr>
+</table>                                                                        |
+
+**Environment compatibility** 
 <table>
 <tbody>
 <tr class="odd">
-<td>Operating systems</td>
-<td><p>For information on tested operating systems, see <a href="https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs">Tested Operating Systems and JDKs</a> .</p></td>
+<th>Operating systems</th>
+<td><p>For information on tested operating systems, see <a href="../../setup/environment-compatibility#tested-operating-systems-and-jdks">Tested Operating Systems and JDKs</a> .</p></td>
 </tr>
 <tr class="even">
-<td>RDBMS</td>
-<td><p>For information on tested DBMSs, see <a href="https://docs.wso2.com/display/compatibility/Tested+DBMSs">Tested DBMSs</a> .</p></td>
+<th>RDBMS</td>
+<td><p>For information on tested DBMSs, see <a href="../../setup/environment-compatibility#tested+dbmss">Tested DBMSs</a> .</p></td>
 </tr>
 <tr class="odd">
-<td>Directory services</td>
+<th>Directory services</td>
 <td><p>Supports Directory Services implementing following LDAP Protocols:</p>
 <ul>
 <li>LDAP v2</li>
 <li>LDAP v3</li>
 </ul>
-<p>For information on tested LDAPs, see <a href="https://docs.wso2.com/display/compatibility/Tested+LDAPs">Tested LDAPs</a> .</p></td>
+<p>For information on tested LDAPs, see <a href="../../setup/environment-compatibility#tested-ldaps">Tested LDAPs</a> .</p></td>
 </tr>
 <tr class="even">
-<td>Java</td>
+<th>Java</td>
 <td>Oracle JDK 1.8 (There’s a <a href="https://bugs.openjdk.java.net/browse/JDK-8189789">known issue</a> with JDK1.8.0_151)</td>
 </tr>
 <tr class="odd">
-<td>Web browsers</td>
-<td><p>For more information on tested web browsers, see <a href="https://docs.wso2.com/display/compatibility/Tested+Web+Browsers">Tested Web Browsers</a> .</p></td>
+<th>Web browsers</td>
+<td><p>For more information on tested web browsers, see <a href="../../setup/environment-compatibility#tested-web+browsers">Tested Web Browsers</a> .</p></td>
```

Comment:
Change this as well -> web+browsers
Please change the plus signs wherever applicable.

Reply:
The respective file was removed from the commit as the file is subject to change—it will be reflected in a subsequent PR.